### PR TITLE
Sync build and release workflows

### PR DESCRIPTION
**Pull request by mikehaller**
The build and release workflows have been updated to **prepare for a technical/milestone build** and testing of the GitHub Actions and their configuration.

Changes:
- Generation of **Software Bill of Material (SBOM)** information in SDPX format. The reports are attached as build artifacts.
- Integration of the **Open Source Software Compliance Review Toolkit (ORT)** into the build process, which checks for compliance of used licenses. The report is attached as build artifact in HTML format. The build will fail if there are any policy violations.
- The BitBake **build process is retried up to three times** in the same build job step. This helps to make the build process more robust and utilizes the already-spent build time as much as possible, e.g. in cases where the lookup of remote sstate-cache is failing due to cache misses or sporadic network failures.
- The limitations of standard GitHub Runners makes it necessary to **externalize the downloads cache and the sstate-cache** to a remote mirror. Locally, also the GitHub Caches are being used to further decrease the necessary build time. The builds for individual target machines are separated jobs, so that they can finish within the 6 hours time limit per job in a workflow.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -232,7 +215,6 @@ jobs:
           CHANGELOG.md \
           README.md \
           3rdparty-licenses.tar.gz \
-          3rdparty-spdx.tar.gz \
           examples
         time tar --append --verbose --dereference --file ${ARC} -C ./build/tmp/deploy/images/raspberrypi4-64/ \
           Image-raspberrypi4-64.bin \
```
Comment (stlachev) [inline .github/workflows/build.yml:220]: You can add here: sdv-image-all-raspberrypi4-64.wic.bmap

**Review by stlachev — APPROVED** [review]
LGTM

File: .github/workflows/build.yml (hunk 1 above)
Comment (stlachev) [inline .github/workflows/build.yml:220]: You can add here: sdv-image-all-raspberrypi4-64.wic.bmap

[the remark this answers is not in the document]
Comment (mikehaller) [reply .github/workflows/build.yml:220]: Done, thanks for the hint

**Pull request merged**